src/pages/edge/adobe-journey-optimizer-decisioning/optimize-configuration-view.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Optimize Setup Validation view
+description: An overview of how the Optimize datastream and sdk setup can be validated.
+keywords:
+- Datastream Validation
+- SDK Validation
+---
+
+# Optimize Setup Validation view
```

Comment:
Let's use the same names as we see in Assurance UI. `Decisioning Validation view`